veggieseasons/lib/widgets/settings_group.dart

```diff
@@ -69,7 +69,7 @@ class SettingsGroup extends StatelessWidget {
     this.header,
     this.footer,
   })  : assert(items != null),
-        assert(items.length > 0);
+        assert(items.isNotEmpty);
```

Comment:
Do people use these properties (`isEmpty`, `isNotEmpty`) consistently? I have nothing more than personal taste to back up my opinion, but for some reason I don't like them.

Reply:
I prefer them as the code reads better, less decoding overhead. As to usage, the `prefer_is_empty` and `prefer_is_not_empty` lint checks will educate and encourage developers. This is why I'm on my add lint checks to everything spree, I witnessed how much value the Go community got out of having one blessed code style.